src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -1,10 +1,20 @@
+@import 'src/styles/common';
+
 .proteinSummary {
   margin-left: 178px;
   width: 695px;
 
   > div {
     padding: 10px 0 10px 0;
   }
+
+  .keyline {
```

Comment:
probably no need to nest it inside the `.proteinSummary` class? Nesting will result in the `.proteinSummary .keyline` CSS rule; while the simple `.keyline` rule should be specific enough